docs/reference/commandline/network_create.md

```diff
@@ -14,18 +14,111 @@ parent = "smn_cli"
 
     Creates a new network with a name specified by the user
 
-      -d, --driver=      Driver to manage the Network
-      --help=false       Print usage
+    --aux-address=map[]      Auxiliary ipv4 or ipv6 addresses used by network driver
+    -d --driver=DRIVER       Driver to manage the Network bridge or overlay. The default is bridge.
+    --gateway=[]             ipv4 or ipv6 Gateway for the master subnet
+    --help=false             Print usage
+    --ip-range=[]            Allocate container ip from a sub-range
+    --ipam-driver=default    IP Address Management Driver
+    -o --opt=map[]           Set custom network plugin options
+    --subnet=[]              Subnet in CIDR format that represents a network segment
 
-Creates a new network that containers can connect to. If the driver supports multi-host networking, the created network will be made available across all the hosts in the cluster. Daemon will do its best to identify network name conflicts. But its the users responsibility to make sure network name is unique across the cluster. You create a network and then configure the container to use it, for example:
+Creates a new network. The `DRIVER` accepts `bridge` or `overlay` which are the built-in network drivers. If you have installed a third party or your own custom network driver you can specify that `DRIVER` here also. If you don't specify the `--driver` option, the command automatically creates a `bridge` network for you.
 
+When you install Docker Engine it creates a `bridge` network automatically.
+This network corresponds to the `docker0` bridge that Engine has traditionally
+relied on. When you `run` a new container it automatically launches on this
+bridge network. You cannot remove this default bridge network but you can create
+new ones using the `network create` command.
+
+```bash
+$ docker network create -d bridge my-bridge-network
+```
+
+Bridge networks are isolated networks on a single Engine installation. If you
+want to create a network that spans multiple Docker hosts each running an
+Engine, you must create an `overlay` network. Unlike `bridge` networks overlay
+networks require some pre-existing conditions before you can create one. These
+conditions are:
+
+* Access to a key-value store. Engine supports Consul, Etcd, Zookeeper (Distributed store), and BoltDB (Local store) key-value stores.
```

Comment:
I dont think we should call out BoltDB here. It is an internal implementation for local-store and is not user-configurable.